manim/mobject/mobject.py

```diff
@@ -1687,6 +1711,7 @@ def get_corner(self, direction):
         return self.get_critical_point(direction)
 
     def get_center(self):
+        """Get center coordinates"""
```

Comment:
@jsonvillanueva : you were suggesting "Center coordinate of the :class:`~.Mobject`." But I think Get center coordinates will do

Reply:
Yes, this is fine. There's no *real* need to mention Mobject here, but if you do, it's better to have it linked/clickable